webapp/components/path-part.html

```diff
@@ -0,0 +1,72 @@
+<!--
+Copyright 2017 The WPT Dashboard Project. All rights reserved.
+Use of this source code is governed by a BSD-style license that can be
+found in the LICENSE file.
+-->
+
+<link rel="import" href="../bower_components/polymer/polymer-element.html">
+
+<!--
+`<path-part>` is a stateless component for displaying part of a test path.
+-->
+<dom-module id="path-part">
+  <template>
+    <style>
+      .dir-path { font-weight: bold; }
```

Comment:
Where is `.file-path`?

Reply:
It's 'not bold' so doesn't have any extra styles.